articles/142_idl.md

```diff
@@ -32,6 +32,14 @@ Further it describes how the interfaces are used to generate code in C, C++ and
 
 ROS 2 supports a subset of the [OMG IDL 4.2 specification](https://www.omg.org/spec/IDL/4.2).
 
+### Lexical Conventions
+
+#### 7.2.2 Comments
+
+#### 7.2.3 Identifiers
+
+#### 7.2.6 Literals
+
```

Comment:
@dirk-thomas are these placeholders for future additions?

Reply:
These are only enumerating the sections of the IDL spec which are supported.

Reply:
I see. Would it make sense to state above that what follows is an enumeration of supported IDL spec sections? It wasn't obvious to me at first sight, the fact that these are the only "empty" sections didn't help inferring it either.

Reply:
I added a short text to each section instead: 826f0119a11b1420db7d1020fc5561b1c284f6b4.